enhancements/update/cvo-techpreview-manifests.md

```diff
@@ -0,0 +1,139 @@
+---
+title: CVO TechPreview Manifests
+authors:
+  - "@deads2k"
+reviewers:
+  - "@wking"
+approvers:
+  - "@sdodson"
+  - "@wking"
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - @sttts
+creation-date: 2021-11-15
+last-updated: 2021-11-15
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - slack thread somewhere
+see-also:
+replaces:
+superseded-by:
+---
+
+# CVO TechPreview Manifests
+
+## Summary
+
+[TechPreviewNoUpgrade](https://github.com/openshift/api/blob/be1be0e89115702f8b508d351c4f5c9a16e5ae95/config/v1/types_feature.go#L32-L34)
+is the canonical way to enable tech preview features.
+It is the setting that all operators use to enable Tech Preview features.
+The CVO will honor this setting on a per-manifest basis, so that tech preview manifests are never present on
+non-tech preview clusters.  
+
+## Motivation
+
+When introducing tech preview operators, the current state of the art (before this enhancement) is to have the 
```

Comment:
nit: here and elsewhere, [trailing whitespace][1].

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_enhancements/957/pull-ci-openshift-enhancements-master-markdownlint/1460316055279767552#1:build-log.txt%3A18

Reply:
> nit: here and elsewhere, [trailing whitespace](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_enhancements/957/pull-ci-openshift-enhancements-master-markdownlint/1460316055279767552#1:build-log.txt%3A18).

imagine all those wasted bytes!